content/08-mongodb/02-connecting-to-mongodb.mdx

```diff
@@ -223,3 +223,52 @@ Knowing how to connect to various MongoDB instances is vital as you start to wor
 If you are using [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) with MongoDB, you can use the [MongoDB connector (currently in preview)](https://www.prisma.io/docs/concepts/database-connectors/mongodb) to connect and manage your data.
 
 </PrismaOutlinks>
+
+## FAQ
+
+<details><summary>What happens when a MongoDB connection is refused?</summary>
+
+Whether connecting to [MongoDB](https://docs.mongodb.com/guides/server/drivers/) locally or remotely, a connection failure will result in an error message from MongoDB. The most common sources of a connection error are an incorrect username or password, attempting to connect with a non-whitelisted IP address, or potentially too many connections to the database.
+
+You can read more about connecting to [MongoDB](https://www.prisma.io/dataguide/mongodb/connecting-to-mongodb) database to ensure you have all of the necessary pieces.
+
+</details>
+
+<details><summary>How do you connect to MongoDB remotely?</summary>
+
+To connect to a [remote MongoDB database](https://www.prisma.io/dataguide/mongodb/connecting-to-mongodb#connecting-to-a-remote-database), you have to include additional details when using the `mongo` shell like `--host` and potentially `--port`.
+
+The basic structure of the command will look something like:
+
+```
+mongo --host <mongo_server_address> --port <mongo_server_port> --user <mongo_username> --password
+```
+
```

Comment:
It might be helpful at this point to mention that the server has to be configured to accept external connections.

Reply:
Good call. I'll find the documentation and link